app.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+
+const { getCategories } = require("./controllers/games-controllers");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/categories", getCategories);
+
+app.all("/*", (req, res) => {
+    res.status(404).send({ msg: "Route not found" });
+});
+
+// PSQL Error Handling
+app.use((err, req, res, next) => {
+    if (err.code === "22P02") {
+        res.status(400).send({ msg: "Bad Request" });
```

Comment:
This error handler will never be called in the current state of the code, and it's not currently being tested. It is good code, however it's not relevant to the project at this stage, so it should be taken out for now and later on re-introduced.